src/app/space/add-app-overlay/add-app-overlay.component.less

```diff
@@ -102,7 +117,14 @@ input,
               height: 100px;
             }
           }
+          &-create-app-image {
+            height: 80px;
```

Comment:
You can set these images to the same height - they both line up the same with the left border when the same height.